bepasty_cli/cli.py

```diff
@@ -55,10 +59,7 @@ def main(token, fileobj, fname, url, ftype):
         stdin = False
 
     else:
-        #  tried to write stdin in tempfile and run guess_file, but guess_file
-        #  only seem to look on the file extension ...
-        # tmpfile = NamedTemporaryFile(delete=False)
-        fileobj = click.get_binary_stream('stdin')
+        fileobj = BytesIO(click.get_binary_stream('stdin').read())
```

Comment:
... the size of your memory. the general use case was uploading small pastes
you cannot seek in stdin unfortunately and tmpfile may be the right thing to do here. 

Still, the problem persists - where do you store big files? /tmp may be tmpfs and map to memory again.

The general use case was to upload snippets generated by other tools like `ps` or netstat.